codestodians/komlie-api
# [Bug] Get quantity for a particular user

#### What does this PR do?
- Fixes a bug observed on the frontend; multiple cart_product items being returned, thus preventing cart render when multiple users have items in cart

#### How should this be manually tested?
- Checkout into this branch and then try to add items to cart for different users. You should find that it works, unlike before when only one user successfully had cart items without an error.

#### Any background context you want to provide?
- While trying to add a new cart item for a newly created user on the frontend, I observed a multiple queryset error in the console. The crux of the matter was that it was impossible to get quantity when a list consists of more than one item. Hence, this story's aim is to increase the filtering parameters to get only one item and thus its quantity.
